ckanext/scheming/templates/scheming/display_snippets/datetime.html

```diff
@@ -0,0 +1,2 @@
+{% set url_parts = data[field.field_name].split() %}
+{{ url_parts[0] }} {{ url_parts[1] }}
```

Comment:
`{{ data[field.field_name] }}` is the same as these two lines, right? which is also the same as just using the existing "text.html" display_snippet